Assets/HoloToolkit/Input/Scripts/Voice/KeywordManager.cs

```diff
@@ -149,6 +156,10 @@ public void StartKeywordRecognizer()
             {
                 keywordRecognizer.Start();
             }
+            else
```

Comment:
I wonder if this would be simpler if the paradigm was 'keyBindingsAlways' instead of 'keyBindingsOnly'.  and wrap keyBindingsAlways in #if UNITY_EDITOR to be true and #else to be false.